pkg/metrics/queue.go

```diff
@@ -105,80 +107,83 @@ func InitQueueMetrics(name string) CoreQueueMetrics {
 func (m *QueueMetrics) Reset() {
 	m.appMetricsLabel.Reset()
 	m.appMetricsSubsystem.Reset()
-	m.ResourceMetricsLabel.Reset()
-	m.ResourceMetricsSubsystem.Reset()
+	m.resourceMetricsLabel.Reset()
+	m.resourceMetricsSubsystem.Reset()
+}
+
+func (m *QueueMetrics) IncQueueApplications(state string) {
```

Comment:
Thanks for the new rev @WilliamTom 

It LGTM except for one very nit: util method (this and other similar ones) that’s only used within the class is private method, use lower case for the first letter of the method name. Can wait for other comments if there are any. Thanks.